pkg/apis/registries/v1alpha1/replication_types.go

```diff
@@ -47,14 +47,14 @@ type ReplicationSpec struct {
 	DestNamespace string `json:"dest_namespace,omitempty"`
 
 	// Source Registry
-	// This uses the registriesv1alpha1 custom Registry specification
+	// Reference to a registry cr
 	// +optional
-	SrcRegistry *RegistrySpec `json:"src_registry,omitempty"`
+	SrcRegistry *corev1.LocalObjectReference `json:"src_registry,omitempty"`
```

Comment:
Sadly, this breaks some tests in `pkg/controller/replication`.
See: https://github.com/mittwald/harbor-operator/pull/12/checks?check_run_id=677320634

Reply:
already fixed. see https://github.com/mittwald/harbor-operator/pull/12/commits/792607bfbd060ca046a46889745b77ccfbd464fc